### PR TITLE
Update doc links

### PR DESCRIPTION
**Pull request by phallobst**

### CODE REVIEW

File: entries/h/hootsuite.com.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
       "totp",
       "email"
     ],
-    "documentation": "https://help.hootsuite.com/hc/en-us/articles/204595950",
+    "documentation": "https://help.hootsuite.com/hc/de/articles/1260804307989",
```
Comment (hkamran80) [inline entries/h/hootsuite.com.json]:
```suggestion
    "documentation": "https://help.hootsuite.com/hc/en-us/articles/1260804307989",
```

File: entries/g/growingio.com.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "tfa": [
       "totp"
     ],
-    "documentation": "https://docs.growingio.com/docs/product-manual/sysmanage/personal/security#kai-qi-er-ci-yan-zheng",
+    "documentation": "https://docs.growingio.com/v3/product-manual/personal/security",
```
Comment (hkamran80) [inline entries/g/growingio.com.json:7]: I'm getting a timeout error when I go to this link. Is it happening for anyone else?

File: entries/u/ukraine.com.ua.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
     "custom-software": [
       "Telegram Bot"
     ],
-    "documentation": "https://www.ukraine.com.ua/en/wiki/account/security/otp/",
+    "documentation": "https://www.ukraine.com.ua/uk/wiki/account/security/otp/",
```
Comment (hkamran80) [inline entries/u/ukraine.com.ua.json]: I'm also getting a timeout error here.

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/g/growingio.com.json (hunk 2 above)
Comment (Carlgo11) [reply entries/g/growingio.com.json:7]: Maybe you're blacklisted by the CCP :laughing: 
Works for me

File: entries/u/ukraine.com.ua.json (hunk 3 above)
Comment (Carlgo11) [reply entries/u/ukraine.com.ua.json]: Loads for me but the content is in Ukrainian.

Comment (phallobst) [reply entries/u/ukraine.com.ua.json]: The English pages seem to be gone. :disappointed:

Comment (hkamran80) [reply entries/u/ukraine.com.ua.json:13]: Should we set it to the Ukranian version then?

Comment (hkamran80) [reply entries/u/ukraine.com.ua.json]: What's our course of action then?

Comment (phallobst) [reply entries/u/ukraine.com.ua.json]: There are many doc pages that are not in English, and nothing says they have to be.

Comment (hkamran80) [reply entries/u/ukraine.com.ua.json]: Should we keep this URL or remove the language code from it?

Comment (phallobst) [reply entries/u/ukraine.com.ua.json]: done

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**